### PR TITLE
Story fixes

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
This will fix #1497

**Comment by sebastianbenz** [comment]
...and it also fixes #1485

**Comment by sebastianbenz** [comment]
Sorry for putting so much into a single PR.

**Comment by sebastianbenz** [comment]
Oh - and also fixes #1433

File: tasks/compile-example.js [hunk 1]
```diff
@@ -297,6 +297,7 @@ module.exports = function(config, indexPath, updateTimestamp) {
       if (document.isAmpStory) {
         generateStoryPreviewEmbed(stream, example, args, {
```
Comment (ithinkihaveacat) [inline tasks/compile-example.js]: The `generateStoryPreviewEmbed` function seems to have only 2 arguments? Am I reading that right?

File: templates/stories/page-switch.html [hunk 2]
```diff
@@ -1,22 +1,72 @@
...
+// Enable story experiment
+enableExperiment();
+
+function enableExperiment() {
+  var expCookie = getCookie(COOKIE_AMP_EXP);
+  if (expCookie) {
+    if (expCookie.indexOf(AMP_STORY) != -1) {
+      return;
+    } else {
+      expCookie += '%2C' + AMP_STORY;
```
Comment (ithinkihaveacat) [inline templates/stories/page-switch.html]: Scattering some `encodeURIComponent()` and `decodeURIComponent()` through the `setCookie()` and `getCookie()` functions might eliminate the need for `%2C`. Or replace `%2C` with `encodeURIComponent(' ')`. Depends how "helpful" the helper functions are supposed to be I guess.

File: templates/stories/page-switch.html [hunk 3]
```diff
@@ -1,22 +1,72 @@
...
     }
-  });
-  story.dispatchEvent(event);
-});
+  } else {
+    expCookie = AMP_STORY;
+  }
+  setCookie(COOKIE_AMP_EXP, expCookie, 356);
```
[the remark this answers is not in the document]
Comment (sebastianbenz) [reply templates/stories/page-switch.html]: extracted a constant, one year is as good as any other value I assume...

File: tasks/compile-example.js (hunk 1 above)
Comment (sebastianbenz) [reply tasks/compile-example.js]: fixed

File: templates/stories/page-switch.html (hunk 2 above)
Comment (sebastianbenz) [reply templates/stories/page-switch.html]: they're only used in a single place, so I didn't bother. But you're right - fixed it.

**Comment by sebastianbenz** [comment]
Adressed the comments. Changed also the way the api is served on localhost. Previews should now work properly there as well. PTAL

**Comment by ithinkihaveacat** [comment]
Tests locally just fine & LGTM!

**Comment by sebastianbenz** [comment]
Thanks!

**Pull request merged**